website/versioned_docs/version-29.0/UpgradingToJest29.md

```diff
@@ -36,7 +36,13 @@ If you want to keep the old behavior, you can set the `snapshotFormat` property
 
 ## JSDOM upgrade
 
-`jest-environment-jsdom` has upgraded `jsdom` from v19 to v20. Note that _if_ you use `jest-environment-jsdom`, the minimum TypeScript version is set to `4.5`.
+`jest-environment-jsdom` has upgraded `jsdom` from v19 to v20.
+
+:::info
+
+_If_ you use `jest-environment-jsdom`, the minimum TypeScript version is set to `4.5`.
```

Comment:
Italic looks unnecessary, if the text is moved to admonition box. I mean, the box already gives that _if_ feeling.

```suggestion
If you use `jest-environment-jsdom`, the minimum TypeScript version is set to `4.5`.
```